config/manager/controller_manager_config.yaml

```diff
@@ -8,4 +8,4 @@ webhook:
   port: 9443
 leaderElection:
   leaderElect: true
-  resourceName: f0fb677d.authorino.3scale.net
+  resourceName: aac3a15d.authorino.kuadrant.net
```

Comment:
kuadrant.io?

Reply:
Also mentioned in https://github.com/Kuadrant/authorino-operator/pull/1#discussion_r741964144